homework5/task_1.py

```diff
@@ -1,4 +1,16 @@
 # Напишите программу, удаляющую из текста все слова, в которых присутствуют все буквы "а", "б", "в"
 
-print('Редактированный текст:', " ".join(filter(
-    lambda x: 'а' and 'в' and 'б' not in x, input('Введите текст: ').split())))
+import string
+
+text = input('Введите текст: ').split()
+
+lst = []
+for i in (text):
+    if 'а' not in i or 'б' not in i or 'в' not in i:
+        lst.append(i)
+    else: 
+        for j in (list(i) [-1]):
+            if j in string.punctuation:
+                lst.append(j)
```

Comment:
```suggestion
for i in text:
    if not ('а' in i and 'б' in i and 'в' in i):
        lst.append(i)
```
Так ведь намного проще?